### ISSUE TITLE
what is the difference between manipulation_dynamic_learning.json and manipulaion_task_learning.json

### ISSUE THREAD

**Issue by 945716994** [wangzizhao/CausalDynamicsLearning#3]
Did in "manipulation_dynamic_learning" just learn a transition model? And not specifiy to a task?
When I change the "env_name" to "Causal" in polcy_params.json just like in "manipulation_dynamic_learning".json, and run the main_policy.py , I got a some error:  **"AttributeError: 'Causal' object has no attribute 'shape'"**

Can you explain the two file's mean?(xxxx_dynamic_xx.json, xxx_task_xxx.json)
Thanks!

**Comment by wangzizhao** [comment]
Yes, as you guess, xxxx_dynamic_xx.json only learns a dynamics model that's not specific to a task.
Then xxx_task_xxx.json load and freeze the dynamics model (the load path is specified by training_params.load_inference) and learns to solve a specific task.
